jekyll-coffeescript.gemspec

```diff
@@ -16,14 +16,14 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR).grep(%r!(lib/)!)
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.3.0"
+  spec.required_ruby_version = ">= 2.4.0"
 
   spec.add_runtime_dependency "coffee-script", "~> 2.2"
   spec.add_runtime_dependency "coffee-script-source", "~> 1.12"
 
-  spec.add_development_dependency "bundler", "~> 1.5"
-  spec.add_development_dependency "jekyll", ENV["JEKYLL_VERSION"] ? "~> #{ENV["JEKYLL_VERSION"]}" : ">= 2.0"
+  spec.add_development_dependency "bundler"
+  spec.add_development_dependency "jekyll", ENV["JEKYLL_VERSION"] ? "~> #{ENV["JEKYLL_VERSION"]}" : ">= 3.0"
```

Comment:
This looks like a breaking-change to me. If we are to ship this in `v1.2.0`, it is going to break legacy sites on Jekyll 2.x but with this plugin set to `~> 1.0`..

/cc @mattr- Whatsay?

Reply:
It's been a while since we no longer test Jekyll 2.x in our CI, why should we bother now with legacy?

Reply:
One of the things that semantic version numbers do is communicate intent. We should bump the major version to signal the intent that this will be backwards compatible in some way. In this case, that's actually no longer supporting Jekyll 2. This change may not _actually_ be breaking anything but it provides feedback that something _may_ break, and that's just as important, IMO.